documentation/api/development.md

```diff
@@ -0,0 +1,72 @@
+# Development
+
+This application is dockerized. Take a look at [Dockerfile](../../api/Dockerfile) to see how it works.
+
+A very simple [docker-compose.yml](../../docker-compose.yml) has been included to support local development and deployment.
+
+## Prerequisites
+
+**Note:** Run everything from within the `/api` folder:
+
+1. Install the version of Python specified in [pyproject.toml](../../api/pyproject.toml)
+   [pyenv](https://github.com/pyenv/pyenv#installation) is one popular option for installing Python,
+   or [asdf](https://asdf-vm.com/).
```

Comment:
maybe add more context to asdf, ie. `and asdf is another popular tool for managing tool versions for more than just python.`